app/views/helpers/address.scala.html

```diff
@@ -16,112 +16,115 @@
-                '_label -> Messages("label.address.postcode"),
-                '_labelClass -> "form-label-bold",
-                'class -> "form-control form-control-1-4",
-                '_error -> form.error("address.postcode"),
-                '_value -> addressData.fold("")(_.postcode)
-            )
-
-            <a href="#" class="lookupAddress hide-if-no-js">@messages("addressLookup.lookup")</a>
+<div class="address--fields show-if-no-js" @if(addressData.isEmpty | selectedAddress.nonEmpty){style="display: none"} else {style="display: block;"}>
+
+
+            @GovukInput(Input(
+            id = "address.line1",
+            name = "address.line1",
+            label = Label(content = HtmlContent(messages(s"label.$fieldName")), classes = "govuk-label--m" ),
+            value =  form("address.line1").value.orElse(Some(addressData.fold("")(_.line1))),
+            errorMessage = form("address.line1").error.map(err => ErrorMessage(content = Text(messages(err.message, err.args:_*)))),
+            classes = "govuk-input govuk-!-width-one-third"
+            ))
+
+
+            @GovukInput(Input(
+
+            id = "address.line2",
+            name = "address.line2",
+            value =  form("address.line2").value.orElse(Some(addressData.fold("")(_.line2))),
+            errorMessage = form("address.line2").error.map(err => ErrorMessage(content = Text(messages(err.message, err.args:_*)))),
+            classes = "govuk-input govuk-!-width-one-third"
+            ))
+
+            @GovukInput(Input(
+            id = "address.line3",
+            name = "address.line3",
+            value =  form("address.line3").value.orElse(Some(addressData.fold("")(_.line3))),
+            errorMessage = form("address.line3").error.map(err => ErrorMessage(content = Text(messages(err.message, err.args:_*)))),
+            classes = "govuk-input govuk-!-width-one-third"
+            ))
+
+
+            @GovukInput(Input(
+            id = "address.line4",
+            name = "address.line4",
+            value =  form("address.line4").value.orElse(Some(addressData.fold("")(_.line4))),
+            errorMessage = form("address.line4").error.map(err => ErrorMessage(content = Text(messages(err.message, err.args:_*)))),
+            classes = "govuk-input govuk-!-width-one-third"
+            ))
+
+
+            @GovukInput(Input(
+            id = "address.postcode",
+            name = "address.postcode",
+            label = Label(content = HtmlContent(messages("label.address.postcode")), classes = "govuk-label--m" ),
+            value =  form("address.postcode").value.orElse(Some(addressData.fold("")(_.postcode))),
+            errorMessage = form("address.postcode").error.map(err => ErrorMessage(content = Text(messages(err.message, err.args:_*)))),
+            classes = "govuk-input govuk-!-width-one-third"
+            ))
+
+            <a href="#" class="govuk-link lookupAddress hide-if-no-js">@messages("addressLookup.lookup")</a>
```

Comment:
why not use a `govukButton` here too?

Reply:
@habeeb01 thats not a button its a link

Reply:
yeah - this is just a link in the original

Reply:
sorry i thought i saw a `govuk-button` class (my bad, i read it wrongly)